modules/ROOT/pages/_partials/rn-new-and-changed-ref.adoc

```diff
@@ -38,4 +38,4 @@ This release includes a second Red Hat Single Sign-On instance for instance to d
 
 == Adding {walkthrough-name}
```

Comment:
Think we should change this to 

== {walkthrough-name}s

NOTE: Walkthroughs are now known as {walkthrough-name}s